src/ubuntu/16.04/cross-arm64/hooks/post-build

```diff
@@ -0,0 +1 @@
+../../../build-scripts/build-rootfs-cleanup.sh
```

Comment:
2 nits:

1. `pre-build` uses `$(dirname ${BASH_SOURCE[0]})`, here it doesn't.  How come?
2. `pre-build` has `#!/usr/bin/env bash`, this doesn't.

Reply:
This is a file link to a shared cleanup file.